contracts/RewardsPool.sol

```diff
@@ -309,6 +316,9 @@ contract RewardsPool is IRewardsPool, Initializable, OwnableUpgradeable, Constan
     /// @notice we will choose the following if `stakeRatio > feeRatio`:
     /// @notice `reward * stakeRatio / e^(alpha * (ln(stakeRatio / feeRatio)))`
     function _cobbDouglas(uint256 reward, uint256 myLabor, uint256 myStake, uint256 totalStake) private view returns (uint256) {
+        if (myLabor == 0 || myStake == 0) {
```

Comment:
This seems unnecessary as `feeRatio` or `stakeRatio` will be zero when `myLabor` or `myStake` is zero.

Otherwise can keep this check and remove
```
       if (feeRatio == 0 || stakeRatio == 0) {
            return 0;
        }
```

Reply:
without this there the tx will fail, some place nested div 0

Reply:
In that case, should this place need change to:
```
if (reward == 0 || totalStake == 0) {
 return 0
}
```

Reply:
The current case is `myLabor` and `myStake` could be 0 (for unregistered indexer) while reward and totalStake > 0

Reply:
You are right though, even reward can not be 0, totalStake can be. I just added totalStake to the check